modules/apps/map/map-openstreetmap/src/main/resources/META-INF/resources/js/MapOpenStreetMap.es.js

```diff
@@ -12,33 +12,48 @@
  * details.
  */
 
-import MapBase from '@liferay/map-common/js/MapBase.es';
-import {Config} from 'metal-state';
+import MapBase from '@liferay/map-common/js/MapBase';
 
 import OpenStreetMapDialog from './OpenStreetMapDialog';
 import OpenStreetMapGeoJSON from './OpenStreetMapGeoJSON';
 import OpenStreetMapGeocoder from './OpenStreetMapGeocoder';
 import OpenStreetMapMarker from './OpenStreetMapMarker';
 
+const defaultTileURI = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+
 /**
  * MapOpenStreetMap
  * @review
  */
 class MapOpenStreetMap extends MapBase {
+	get tileURI() {
+		return this._STATE_.tileURI;
+	}
+
+	set tileURI(tileURI) {
+		this._STATE_.tileURI = tileURI;
+	}
 
 	/**
 	 * Creates a new map using OpenStreetMap's API
 	 * @param  {Array} args List of arguments to be passed to State
 	 * @review
 	 */
-	constructor(...args) {
+	constructor(args) {
 		MapBase.DialogImpl = OpenStreetMapDialog;
 		MapBase.GeocoderImpl = OpenStreetMapGeocoder;
 		MapBase.GeoJSONImpl = OpenStreetMapGeoJSON;
 		MapBase.MarkerImpl = OpenStreetMapMarker;
 		MapBase.SearchImpl = null;
 
-		super(...args);
+		super(args);
+
+		this._STATE_ = {};
+
+		const {boundingBox, tileURI = defaultTileURI} = args;
+
+		this.tileURI = tileURI;
+		this.boundingBox = boundingBox;
```

Comment:
Just to keep consistent with your class above, you can set them as default in `this._STATE_ = {}` and remove these

Reply:
👍